### DISCUSSION TITLE
Welcome to Wops Discussions!

### DISCUSSION THREAD

**Discussion by redocmath** [Wopslang/Wops#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by redocmath** [comment]
Yeah, that's right. I just used the boilerplate template. :D

**Reply by dddddd1234-creator** [reply]
Nice Job

**Reply by redocmath** [reply]
lol
